setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+UTILS_CV = "utils_cv"  # Utility folder name
+# UTILS_CV_PATH = path.join(path.abspath(path.dirname(__file__)), UTILS_CV)
+# README = path.join(UTILS_CV_PATH, "README.md")
+README = path.join(UTILS_CV, "README.md")
+exec(open(path.join(UTILS_CV, "__init__.py")).read())
+
+with open(README, encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="utils_cv",
+    version=__version__,
+    description="Computer Vision Utilities",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/ComputerVision",
+    project_urls={
+        "Bug Tracker": "https://github.com/microsoft/ComputerVision/issues",
+        "Source Code": "https://github.com/microsoft/ComputerVision",
+        "Documentation": "https://github.com/microsoft/ComputerVision",
+    },
+    author="CVDev Team at Microsoft",
+    classifiers=[
+        "Development Status :: 2-Pre-Alpha",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    include_package_data=True,
+    # Not fully test the dependency coverage, please make sure to sync
+    # with environment.yml
+    install_requires=[
+        "azureml-sdk[notebooks,contrib]>=1.0.30",  # requires ipykernel, papermill, jupyter-core, jupyter-client
+        "bqplot",
+        "fastai==1.0.48",  # requires pytorch, torchvision, nvidia-ml-py3
```

Comment:
the main difference I see with what we are doing in recommenders or nlp is that there we are installing different versions for cpu and gpu, so our installation process is more complicated. Your installation dependencies are much simpler. I wonder what happens if you install this library in a new environment on a GPU machine, does it automatically install pytorch gpu?

Reply:
Yeah, currently it is much simpler for CVBP than Reco.  I am not sure if we need to separate it into CPU version, GPU version and Spark version, such as `utils_reco_cpu`, `utils_reco_gpu` and `utils_reco_spark`, and do similar thing for `utils_cv`.  That needs to be fully tested.  The only thing I know for PyTorch said in [fast.ai installation](https://github.com/fastai/fastai/blob/master/README.md#installation) is 
> Starting with pytorch-1.x you no longer need to install a special pytorch-cpu version. Instead use the normal pytorch and it works with and without GPU. But you can install the cpu build too.

Reply:
ok great

Reply:
Yes. Our code and installation instructions is identical for CPU and GPU targets.